data/recipes/vt_evtx.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "vt_evtx",
+    "description": "Fetches the EVTX for a specific hash, processes it with Plaso and puts it to Timesketch.",
```

Comment:
There's no `TimesketchExporter` here, so could you adjust the description for the time being?